### ISSUE TITLE
[🐞] Could not resolve entry_QwikCityProvider.js when build

### ISSUE THREAD

**Issue by anboo44** [BuilderIO/qwik#3937]
### Which component is affected?

Qwik City (routing)

### Describe the bug

I build project on windows 10 and use command: `pnpm build`. Error log happens: 

```
vite v4.3.2 building for production...
✓ 10 modules transformed.
✓ built in 1.56s
Could not resolve "./entry_QwikCityProvider.js" from "node_modules/.pnpm/@builder.io+qwik-city@0.104.0_@builder.io+qwik@0.104.0/node_modules/@builder.io/qwik-city/index.qwik.mjs"      
file: C:/Projects/frontend/node_modules/.pnpm/@builder.io+qwik-city@0.104.0_@builder.io+qwik@0.104.0/node_modules/@builder.io/qwik-city/index.qwik.mjs
error during build:
RollupError: Could not resolve "./entry_QwikCityProvider.js" from "node_modules/.pnpm/@builder.io+qwik-city@0.104.0_@builder.io+qwik@0.104.0/node_modules/@builder.io/qwik-city/index.qw
ik.mjs"
    at error (file:///C:/Projects/frontend/node_modules/.pnpm/rollup@3.21.0/node_modules/rollup/dist/es/shared/node-entry.js:2125:30)
    at ModuleLoader.handleInvalidResolvedId (file:///C:/Projects/frontend/node_modules/.pnpm/rollup@3.21.0/node_modules/rollup/dist/es/shared/node-entry.js:23913:24)
    at ModuleLoader.resolveDynamicImport (file:///C:/Projects/frontend/node_modules/.pnpm/rollup@3.21.0/node_modules/rollup/dist/es/shared/node-entry.js:23971:58)
    at async file:///C:/Projects/frontend/node_modules/.pnpm/rollup@3.21.0/node_modules/rollup/dist/es/shared/node-entry.js:23860:32
 ELIFECYCLE  Command failed with exit code 1.
```

How to fix it ? 

### Reproduction

No have link

### System Info

```shell
System:
    Windows: 10
    CPU: Intel Core i5 8300H 2.30GHz
    Memory: 4 GB / 16.00 GB
  Binaries:
    Node: 20.0.0
    pnpm: 8.3.1
  Browsers:
    Brave Browser: 111.1.49.120
  npmPackages:
    @builder.io/qwik: 0.104.0 
    @builder.io/qwik-city: ~0.104.0 
    vite: 4.3.2
```

**Comment by adamdbradley** [comment]
Would you be able to try again with version 1.1.1? There recently was a windows path fix around this. Thanks

**Comment by anboo44** [comment]
@adamdbradley The problem is resolved at version 1.1.1. I close this thread. Thank you
